Source/UZKArchive.m

```diff
@@ -923,8 +920,9 @@ - (BOOL)extractBufferedDataFromFile:(NSString *)filePath
             UZKLogError("Error closing file (code %d): %{public}@", err, detail);
             [welf assignError:innerError code:err
                        detail:detail];
-            return;
```

Comment:
@amosavian Why not return after this error?

Reply:
I just wanted to check if there was something I was missing here. I've removed this return statement in my latest push.

Reply:
Let's resolve open conversations, and I'll merge this guy!